src/components/voting/PollEditForm/lang.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+export const prefix = 'PollEditForm'
+
+export default {
+  en: {
+    title: 'Create New Poll',
+    editPoll: 'Edit Poll',
+    newPoll: 'New Poll',
+    pollTitle: 'Headline',
+    pollDescription: 'Description',
+    voteLimit: 'Vote Limit in TIME per option',
+    finishedDate: 'Date',
+    finishedTime: 'Time',
+    addAttachments: 'Attach new',
+    options: 'Options',
+    option: 'Option description',
+    optionIndex: 'Option #%{index}',
+    updatePoll: 'Update Poll',
+    createPoll: 'Create Poll',
```

Comment:
[Question]

At line 10 we have `title: 'Create New Poll',`
Do we really need 'Create Poll' and 'Create New Poll' at the same time?

Reply:
Fixed!